File: SPOJ programs(java)/rectangles.java
import java.util.Scanner;


public class rectangles {

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n=sc.nextInt();
	    
	   
	    int i,r;
	    int s = n;
	    r =(int)Math.sqrt(n);
	    for(i = 2;i<=r;i++)
	    {
	        s = s + (n/i-(i-1));
	    }
	   System.out.println(s);

	}

}

File: SPOJ programs(c)/rectangles.c
#include<stdio.h>
#include<math.h>
int main(void)
{
    int sum,n,r,i;
    scanf("%d",&n);
    r = (int)(sqrt(n));
    sum = n;
    for(i = 2;i<=r;i++)
    {
        sum = sum + (n / i) - (i-1);
    }

    printf("%d",sum);
}
